app/assets/stylesheets/application/_footer.scss

```diff
@@ -0,0 +1,53 @@
+.footer {
+  background: black;
+  color: $gray;
+  display: block;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  padding: 2rem 1rem 5rem 1rem;
+  text-transform: uppercase;
```

Comment:
I'm not a fan of uppercase text, but if the design requires it I'd rather we attach the style to a more specific element (e.g. `.footer a` instead of the whole footer area, or `.featured-action h2` instead of all h2).

Reply:
yup